docs/source/plugins/06_solver_hooks.rst

```diff
@@ -237,6 +237,16 @@ each Liquid-Liquid system plugin hooks. In the sequence, they are presented
 
 .. autofunction:: alfasim_sdk._internal.hook_specs.calculate_liq_liq_shear_force_per_volume
 
+Emulsion Model
+-------------------------------
+
+When a simulation case uses a three-phase flow, the |alfasim|'s simulator allows to select an emulsion model that will
```

Comment:
I'm not sure if this is what you meant, but what about this rephrase:

"When a simulation case uses a three-phase flow model, |alfasim| allows the selection of an emulsion model to calculate emulsion viscosity. The model may be further customized through the implementation of a plugin for relative emulsion viscosity."

Reply:
I think your suggestion is very good, but I want to give the information on how this hook is used, which is `update the layer viscosity related to the emulsion`.

Reply:
Ok, no problem

Reply:
I updated it as we discussed in the RC.